dulwich/contrib/swift.py

```diff
@@ -257,7 +257,7 @@ def swift_auth_v1(self):
         if ret.status_code < 200 or ret.status_code >= 300:
             raise SwiftException(
                 "AUTH v1.0 request failed on "
-                + f"{str(auth_httpclient.get_base_url()) + path} with error code {ret.status_code} ({ret.items()!s})"
+                + f"{self.auth_url} with error code {ret.status_code} ({ret.items()!s})"
```

Comment:
I am not sure if this information is safe to print, I am also unsure why the author of this code used `auth_httpclient.get_base_url` instead of `self.auth_url` and if they are equal

Reply:
Let's optimistically assume this is okay. If we're wrong, we can revert and add a comment (and add a comment)